_posts/2022-05-02-the-na-bug/the-na-bug.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Ooh na na... where are my sodium labs?"
+description: |
+  The NA bug, or, what happens when the same word is used in different contexts.
+author:
+  - name: Chloe Pou-Prom
+    affiliation: DSAA, Unity Health Toronto
+    affiliation_url: https://chartdatascience.ca
+date: 2022-05-02
+output:
+  distill::distill_article:
+    self_contained: false
+categories:
+  - language-R
+  - project-chartwatch
+  - post-miscellaneous
+draft: true
+preview: preview.PNG
```

Comment:
By default, the first figure will show up as the preview image in the index page. 

.... However that only works for figures created by `ggplot2`! All my figures were created in` dygraphs`.

I ended up taking a screenshot for the `dygraphs` preview figure.